apps/web/components/IconTitleCard/IconTitleCard.tsx

```diff
@@ -18,29 +27,56 @@ export const IconTitleCard = ({
   alt,
   href,
   dataTestId,
+  reverseOrder = false,
+  withButton = false,
+  buttonTitle,
 }: IconTitleCardProps) => (
   <FocusableBox
     href={href ?? undefined}
     display="flex"
-    flexDirection="row"
+    justifyContent={reverseOrder ? 'spaceBetween' : 'flexStart'}
+    flexDirection={reverseOrder ? 'rowReverse' : 'row'}
+    padding={reverseOrder ? 3 : 0}
     alignItems="center"
     background="purple100"
     borderColor="purple100"
     borderRadius="large"
     borderWidth="standard"
     height="full"
-    className={styles.container}
+    className={withButton ? styles.containerBig : styles.container}
     color="purple"
     data-testid={dataTestId}
   >
-    <span className={styles.iconContainer}>
-      <img src={imgSrc} alt={alt} className={styles.icon} />
+    <span
+      className={withButton ? styles.iconContainerBig : styles.iconContainer}
+    >
+      <img
+        src={imgSrc}
+        alt={alt}
+        className={withButton ? styles.iconBig : styles.icon}
+      />
     </span>
-    <div className={styles.titleContainer}>
-      <Text variant="h5" color="purple600" truncate>
-        {heading}
-      </Text>
-    </div>
+    <Box display="flex" flexDirection="column" justifyContent="spaceBetween">
+      <div className={styles.titleContainer}>
+        <Text variant={withButton ? 'h4' : 'h5'} color="purple600" truncate>
+          {heading}
+        </Text>
+      </div>
+      {withButton && (
+        <Box marginTop={1}>
+          <Button
+            variant="text"
+            as="span"
+            icon="arrowForward"
+            size="small"
+            colorScheme="purple"
+            nowrap
+          >
+            {buttonTitle ? buttonTitle : 'Skoða nánar'}
```

Comment:
Icelandic hardcoded button text feels off to me, could we maybe display an arrow instead or have the buttonTitle also mandatory for the button to appear?